src/coffee/services/base.coffee

```diff
@@ -556,7 +556,14 @@ class BaseService
 
       # TODO: check other possible acceptable codes (304, ...)
       if 200 <= response.statusCode < 300
-        resolve _.extend responseJson,
+        # FIXME: find a better solution
+        if @constructor.name is 'GraphQLService' and not body.data
+          graphqlError = new GraphQLError 'GraphQL error', _.extend responseJson, body,
+            statusCode: 400
+            originalRequest: originalRequest
+          return reject graphqlError
```

Comment:
@OlegIlyenko I noticed that the endpoint return always `200` even if there are `errors`. Is that by design?